x-mfloat-gprs=double -Wa,-me500x2  flag break cc-gcc-core-pass2

I'm attempting to build a PowerPC cross compiler for the e500x2 PowerPC for baremetal.
I'm using ct-ng 1.23
When I build, I get an error saying:

[ERROR]    checking for suffix of object files... configure: error: in `/home/zwilcox/crosstool-ng/.build/powerpc-unknown-elf/build/build-cc-gcc-core-pass-2/powerpc-unknown-elf/libgcc':

The config.log file in ./build/powerpc-unknown-elf/build/build-cc-gcc-core-pass2/
has:
configure:4273: x86_64-build_pc-linux-gnu-gcc -V >&5
gcc: error: unrecognized command line option '-V'

Attached is config.log, build.log and .config

[build.log](https://github.com/crosstool-ng/crosstool-ng/files/2354837/build.log)
[config.log](https://github.com/crosstool-ng/crosstool-ng/files/2354838/config.log)
[config.txt](https://github.com/crosstool-ng/crosstool-ng/files/2354839/config.txt)

The problem exists with the following:
 -mfloat-gprs=double -Wa,-me500x2
-mfloat-gprs=double
and
-me500x2
The build is successful with those flags removed.

--- a/scripts/build/arch/powerpc.sh
+++ b/scripts/build/arch/powerpc.sh
@@ -12,7 +12,7 @@ CT_DoArchTupleValues () {
             ;;
         spe)
             case "${CT_LIBC}" in
-                none|newlib)    CT_TARGET_SYS="spe";;
+                none|newlib)    CT_TARGET_SYS="elfspe";;
                 *glibc)         CT_TARGET_SYS="gnuspe";;
                 uClibc)         CT_TARGET_SYS="uclibcgnuspe";;
             esac

